conbench/logger.py

```diff
@@ -0,0 +1,94 @@
+"""
+Set up Python's logging infrastructure for the web application.
+
+Acquire root logger instance and populate it with handlers, applying a certain
+log format. This takes effect for Conbench code as well as integrated
+libraries, such as SQLAlchemy
+
+A package or module within Conbench should obtain and use a logger instance in
+the 'conbench' sub-tree of the logging namespace hierarchy, via e.g.
+
+    log = logging.getLogger('conbench.xxx')
+"""
+
+
+import logging
+import logging.handlers
+
+
+def setup(
+    level_stderr="ERROR",
```

Comment:
Right now we have defaults in 3 different places (for example, `level_stderr="ERROR"` here, `level_stderr="DEBUG"` in `__init__.py`, and `LOG_LEVEL_STDERR="INFO"` in the app config). Would you be open to removing the defaults from this internal function and explicitly passing them through in the other two files, so that it's a little clearer which level will be used at what times? Then we don't have to think as hard about "which default level got picked up in this case"

Reply:
>  Would you be open to removing the defaults from this internal function and explicitly passing them through in the other two files, so that it's a little clearer which level will be used at what times?

Yes, totally open. I agree. I also found that this smells a bit. I was basically just copy/pasting code around.